src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -409,9 +371,13 @@ You should see the item created: `Learn AWS AppSync`.
 
 </BlockSwitcher>
 
-You do not have to pass in `createdAt` and `updatedAt` fields. AppSync manages this for you.
+<Callout>
```

Comment:
Put this one in a note callout to keep consistent with one that follows below.

Reply:
sounds good.